src/Chirp.Razor/DBFacade.cs

```diff
@@ -0,0 +1,50 @@
+using System.Data;
+using Microsoft.Data.Sqlite;
+using SQLitePCL;
+
+namespace Chirp.Razor
+{
+    public class DBFacade
+    {
+        public static List<CheepViewModel> Read()
+        {
+            List<CheepViewModel> cheeps = new List<CheepViewModel>();
+            
+            // Initialize SQLite provider ChatGPT help
+            Batteries.Init();
+
+            string sqlDBFilePath = Environment.GetEnvironmentVariable("CHIRPDBPATH");
```

Comment:
There is a build warning here on line 16 to review: "Converting null literal or possible null value to non-nullable type."

Reply:
I will resolve this warning in a commit.